README.md

```diff
@@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
 * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
 * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
 * [Inna Shteinbuk](people/inna-shteinbuk.md)
+* [Jacqueline J.ø Pang](people/jacqueline-pang.md)
```

Comment:
Hi Dan
Yes, you are right  this is a typo.
Do I need to push a 5th try to pull a new request?


Sent from my iPhone

> On 2015年9月2日, at 14:54, "Daniel Doubrovkine (dB.) @dblockdotorg" <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -98,6 +98,7 @@ Read about guest lectures [here](course/guest-lectures.md).
> >  * [Hsiao-Ching (Jean) Lin](people/hsiaoching-lin.md)
> >  * [Huai-Che (Acsa) Lu](people/huaiche-lu.md)
> >  * [Inna Shteinbuk](people/inna-shteinbuk.md)
> > +* [Jacqueline J.ø Pang](people/jacqueline-pang.md)
> Is J.ø a typo or is it really part of your name?
> 
> —
> Reply to this email directly or view it on GitHub.
>